ipython/mechanism_analyzer.ipynb

```diff
@@ -837,50 +806,57 @@
     "    flux = tup[1]\n",
     "    if flux > flux_threshold:\n",
     "        total_flux += flux\n",
-    "        print ''\n",
-    "        print \"**********************************************************************************\"\n",
-    "        if showIntegratedFlux:\n",
-    "            print \"rxn#{0}: {1}: Integrated Flux = {2:.3E} mole \".format(rxn.index, str(rxn), flux) \n",
+    "        print('')\n",
+    "        print(\"**********************************************************************************\")\n",
+    "        if show_integrated_flux:\n",
+    "            print(\"rxn#{0}: {1}: Integrated Flux = {2:.3E} mole \".format(rxn.index, str(rxn), flux) )\n",
     "        else:\n",
-    "            print \"rxn#{0}: {1}: Flux = {2:.3E} mole/cm3/s\".format(rxn.index, str(rxn), flux)\n",
+    "            print(\"rxn#{0}: {1}: Flux = {2:.3E} mole/cm3/s\".format(rxn.index, str(rxn), flux))\n",
     "        display(rxn) \n",
-    "print \"***************************************\"\n",
-    "print ''\n",
-    "if showIntegratedFlux:\n",
-    "    print \"TOTAL integrated flux from h_abs and disp is {0:.3E} mole.\".format(total_flux)\n",
+    "print(\"***************************************\")\n",
+    "print('')\n",
+    "if show_integrated_flux:\n",
+    "    print(\"TOTAL integrated flux from h_abs and disp is {0:.3E} mole.\".format(total_flux))\n",
     "else:\n",
-    "    print \"TOTAL flux from h_abs and disp is {0:.3E} mole/cm3/s.\".format(total_flux)\n"
+    "    print(\"TOTAL flux from h_abs and disp is {0:.3E} mole/cm3/s.\".format(total_flux))\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 2",
+   "display_name": "Python [conda env:rmg_env]",
    "language": "python",
-   "name": "python2"
+   "name": "conda-env-rmg_env-py"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 2
+    "version": 3
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython2",
-   "version": "2.7.12"
+   "pygments_lexer": "ipython3",
+   "version": "3.7.4"
+  },
+  "pycharm": {
+   "stem_cell": {
+    "cell_type": "raw",
+    "metadata": {
+     "collapsed": false
+    },
+    "source": []
+   }
```

Comment:
Are these changes necessary or can they be discarded?

Reply:
They're probably unnecessary. I opened this on in PyCharm to edit so I could use more advanced functionality. I guess it added additional metadata.